PC/launcher.c

```diff
@@ -1248,7 +1248,7 @@ static PYC_MAGIC magic_values[] = {
     { 3390, 3399, L"3.7" },
     { 3400, 3419, L"3.8" },
     { 3420, 3429, L"3.9" },
-    { 3430, 3439, L"3.10" },
+    { 3430, 3440, L"3.10" },
```

Comment:
This is a range. No need to change it. But you need to increase the magic number in `Lib/importlib/_bootstrap_external.py`.

Reply:
Fixed, my bad.